src/libsystemd/sd-bus/bus-control.c

```diff
@@ -164,12 +164,12 @@ static int default_request_name_handler(
 
         case BUS_NAME_EXISTS:
                 log_debug("Requested service name already owned, failing connection.");
-                bus_enter_closing(sd_bus_message_get_bus(m));
+                (void) bus_enter_closing(sd_bus_message_get_bus(m), -EEXIST);
                 return 1;
         }
 
         log_debug("Unexpected response from RequestName(), failing connection.");
-        bus_enter_closing(sd_bus_message_get_bus(m));
+        (void) bus_enter_closing(sd_bus_message_get_bus(m), -ENOANO);
```

Comment:
Is the argument supposed to be positive or negative (or it doesn't matter)? `sd_bus_message_get_errno` returns non-negative.